test/recipes/90-test_store.t

```diff
@@ -152,6 +164,25 @@ indir "store_$$" => sub {
                 ok(!run(app(["openssl", "storeutl",  "-noout", $_])));
             }
         }
+
+	if ($test_src_rsa_files) {
+	    foreach (@src_rsa_files) {
+		my $file = srctop_file($_);
+		my @pubin = $_ =~ m|pub\.pem$| ? ("-pubin") : ();
+
+		ok(run(app(["openssl", "rsa", "-text", "-noout", @pubin,
+			    "-engine", "ossltest", "-inform", "engine",
+			    "-in", "ot:".$file])));
+		ok(run(app(["openssl", "rsa", "-text", "-noout", @pubin,
+			    "-engine", "ossltest", "-inform", "engine",
+			    "-in", "ot:".to_abs_file($file)])));
+		ok(run(app(["openssl", "rsa", "-text", "-noout", @pubin,
+			    "-in", "engine:ossltest:ot:$file"])));
+		ok(run(app(["openssl", "rsa", "-text", "-noout", @pubin,
+			    "-in", "engine:ossltest:ot:".to_abs_file($file)])));
```

Comment:
Given the extent of the option changes in the apps this seems very light testing